src/components/NcChip/NcChip.vue

```diff
@@ -190,7 +190,7 @@ const onClose = () => {
 <style scoped lang="scss">
 .nc-chip {
 	--chip-size: 24px;
-	--chip-radius: calc(var(--chip-size) / 2);
+	--chip-radius: var(--border-radius-small);
```

Comment:
Does not work as this does not exist on Nextcloud 28 and 29, so the fallback to current design is needed.